pipx/main.py

```diff
@@ -590,6 +604,11 @@ def run_pipx_command(args):
             verbose,
             force=args.force,
         )
+    elif args.command == "inject":
+        package_or_url = (
+            args.spec if ("spec" in args and args.spec is not None) else args.dependency
+        )
+        inject(venv_dir, args.dependency, package_or_url, verbose)
```

Comment:
I think this should just be
```suggestion
        inject(venv_dir, args.dependency, verbose)
```